.github/CODEOWNERS

```diff
@@ -110,7 +110,9 @@
 # Contributor Dashboard project.
 /core/controllers/contributor_dashboard*.py @sagangwee
 /core/domain/opportunity*.py @sagangwee
+/core/domain/translation*.py @vojtechjelinek
 /core/domain/voiceover_services*.py @sagangwee
+/core/storage/translation/ @vojtechjelinek
```

Comment:
Please don't include this line. All storage models should be owned by @BenHenning (see entry at the bottom of the codeowners file). It's fine to just delete this line.

Reply:
Done.